engine/src/test/java/org/hibernate/search/test/configuration/mutablefactory/MutableFactoryTest.java

```diff
@@ -152,6 +155,77 @@ public void testAddingClassSimpleAPI() throws Exception {
 		sf.close();
 	}
 
+	@Test
+	public void testAddingClassSimpleAPIwithJMX() throws Exception {
+		SearchIntegrator sf = new SearchIntegratorBuilder().configuration( new SearchConfigurationForTest().addClass( A.class ).addProperty( "hibernate.search.jmx_enabled" , Boolean.TRUE.toString() ).addProperty( "hibernate.search.generate_statistics", Boolean.TRUE.toString() ).addProperty( "com.sun.management.jmxremote", Boolean.TRUE.toString() ) ).buildSearchIntegrator();
+
```

Comment:
right.. I realize our style formatter is a bit pedantic, but it will allow you to break down lines in a more understandable shape.

could you reformat it for better readability?

    SearchIntegrator sf = new SearchIntegratorBuilder()
				.configuration(
						new SearchConfigurationForTest()
							.addClass( A.class )
							.addProperty( "hibernate.search.jmx_enabled" , Boolean.TRUE.toString() )
							.addProperty( "hibernate.search.generate_statistics", Boolean.TRUE.toString() )
							.addProperty( "com.sun.management.jmxremote", Boolean.TRUE.toString() ) )
				.buildSearchIntegrator();